Nos/Views/LikeButton.swift

```diff
@@ -18,8 +22,13 @@ struct LikeButton: View {
     @Environment(\.managedObjectContext) private var viewContext
     @ObservationIgnored @Dependency(\.analytics) private var analytics
 
-    internal init(note: Event) {
+    /// Initializes a LikeButton object.
+    ///
+    /// - Parameter note: Note event to display likes to.
```

Comment:
I think this would be a little more clear:
```suggestion
    /// - Parameter note: The event associated with this Like button.
```